Sources/DateExtensions.swift

```diff
@@ -136,6 +136,30 @@ extension Date {
         }
     }
     
+    /// EZSE: Easy creation of time passed String. Can be Years, Months, days, hours, minutes or seconds. Useful for localization
+    public func timePassed() -> TimePassed {
+        
+        let date = Date()
+        let calendar = Calendar.autoupdatingCurrent
+        let components = (calendar as NSCalendar).components([.year, .month, .day, .hour, .minute, .second], from: self, to: date, options: [])
+        
+        if components.year! >= 1 {
```

Comment:
What if you have to represent something like 1 year and 6 months. IMO it should be both TimePassed.year(1) and TimePassed.month(6). Here it would just round up to 1 year.

Reply:
You're right, it would just round up to the largest time unit, because we retained the original behaviour of the first `-timePassed` extension method. The original method would return `"1 year ago"`, instead of `"1 year and 6 months ago"`. 

On developers' (consumers') perspective, you're still right on what the `-timePassed` method should return.